test/e2e/sagemaker/resources/xgboost_hpojob.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
+kind: HyperParameterTuningJob
+metadata:
+  name: $HPO_JOB_NAME
+spec:
+  hyperParameterTuningJobName: $HPO_JOB_NAME
+  tags:
+    - key: test-key
+      value: test-value
+  hyperParameterTuningJobConfig:
+    strategy: Bayesian
+    hyperParameterTuningJobObjective:
+      type_: Minimize
+      metricName: validation:error
+    resourceLimits:
+      maxNumberOfTrainingJobs: 10
+      maxParallelTrainingJobs: 5
```

Comment:
Please reduce down to 2/1 maybe. We want to reduce costs for anyone that may want to run these integration tests

Reply:
we stop the job right after submitting it so shouldn't matter much 

but ya making the change